src/components/VButton/VButton.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <button
+    class="button"
+    v-on="$listeners"
+  >
+    <slot />
+  </button>
+</template>
+
+<script>
+import t from "vue-types"
+
+export default {
+  props: {
+    text: t.string,
+  }  
+}
+</script>
```

Comment:
Try making this a functional component as currently it does not need any props any more. 🤔 
```suggestion
<template functional>
  <button
    class="button"
    v-on="$listeners"
  >
    <slot />
  </button>
</template>
```